MechJeb2/MechJebModuleNodeEditor.cs

```diff
@@ -229,6 +229,11 @@ protected override void WindowGUI(int windowID)
                     {
                         core.node.ExecuteOneNode(this);
                     }
+					
+		    if (GUILayout.Button(Localizer.Format("Execute next Principia node")))
+		    {
+                        core.node.ExecuteOnePNode(this);
+                    }
```

Comment:
I'm not wild about throwing this button at stock players, or even non-principia users.  This should probably be gated by a check that principia is loaded (there's some code in the guidance controller that starts to do that, it should probably get moved somewhere like VesselState (similar to the FAR checks there), or something like that, and made available so that every module can check if principia is loaded or not.

Reply:
Done! However, the location of the code that check if Principia is loaded remain unchanged. .

Reply:
heh okay bit wonky having it live there but that lets me clean that up later.